core/src/main/java/io/questdb/std/str/Utf8String.java

```diff
@@ -80,18 +88,26 @@ public int intAt(int index) {
         return Unsafe.byteArrayGetInt(bytes, index);
     }
 
+    @Override
+    public long longAt(int offset) {
+        return Unsafe.byteArrayGetLong(bytes, offset);
+    }
+
     @Override
     public boolean isAscii() {
         return ascii;
     }
 
-    public long longAt(int index) {
-        return Unsafe.byteArrayGetLong(bytes, index);
+    @Override
+    public long zeroPaddedSixPrefix() {
+        assert size() > VARCHAR_MAX_BYTES_FULLY_INLINED
+                : String.format("size %,d <= %d", size(), VARCHAR_MAX_BYTES_FULLY_INLINED);
+        return longAt(0) & VARCHAR_INLINED_PREFIX_MASK;
     }
 
     @Override
     public int size() {
-        return bytes.length;
```

Comment:
nit: previous version with `bytes.length` was efficient, no need to copy the array length into a dedicated field